### PR TITLE
Fix `context create`: Use values from template context

### LINKED ISSUE #2976: [mesheryctl] Context: New contexts not created from Template Config
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior
utils/helpers.go has multiple references to Meshery's endpoint and endpoint protocol.

`mesheryctl system context create does not use the , but uses https://github.com/layer5io/meshery/blob/9deb67c837b7de17fc7cfc81fcb47b0dc9cc3816/mesheryctl/pkg/utils/helpers.go#L80-L80, not https://github.com/layer5io/meshery/blob/4cce0a7e1599410bd2d95c24b9185ea6af0abc94/mesheryctl/pkg/utils/helpers.go#L124-L124

#### Desired Behavior
These references should be consolidated.

#### Contributor Resources

- [mesheryctl Contributing Guide](https://github.com/layer5io/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by pottekkat**
Signed-off-by: navendu-pottekkat <navendupottekkat@gmail.com>

**Description**

This PR fixes #2976

Use the template config for setting default values and only change the values if flags are provided.

This PR reduce rewriting some of the values of the constants and re-referencing it.

PS: No changes to the stuff visible to the user.

Create a new context with default settings- 

Screenshot from 2021-05-22 17-40-34

Create a new context with all available flags-

Screenshot from 2021-05-22 17-41-06

See Bob and Wendy-

Screenshot from 2021-05-22 17-41-38

- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Bob is lonely no longer.

File: mesheryctl/internal/cli/root/system/context/create.go [hunk 1]
```diff
@@ -23,19 +23,26 @@ var createContextCmd = &cobra.Command{
...
-			return err
+		if url != "" {
+			err := utils.ValidateURL(url)
+			if err != nil {
+				return err
+			}
+			tempContext.Endpoint = url
+		}
+
+		log.Printf("url: `%s`", tempContext.Endpoint)
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/system/context/create.go:35]: Shall we move this to `-v` or delete?

Comment (navendu-pottekkat) [reply mesheryctl/internal/cli/root/system/context/create.go:35]: Moving this to debug log

**Comment by navendu-pottekkat** [comment]
@leecalcote Done!

**Comment by leecalcote** [comment]
Boom! 💣

**Review by leecalcote — APPROVED** [review]

**Pull request merged**